client/infinispan-key-value-store-hotrod/src/main/java/org/infinispan/api/client/configuration/reactive/Configuration.java

```diff
@@ -0,0 +1,29 @@
+package org.infinispan.api.client.configuration.reactive;
+
+import java.util.List;
+import java.util.function.Supplier;
+
+import org.infinispan.api.ClientConfig;
+import org.infinispan.client.hotrod.FailoverRequestBalancingStrategy;
+import org.infinispan.client.hotrod.ProtocolVersion;
+import org.infinispan.client.hotrod.configuration.ClientIntelligence;
+import org.infinispan.client.hotrod.configuration.ClusterConfiguration;
+import org.infinispan.client.hotrod.configuration.ConnectionPoolConfiguration;
+import org.infinispan.client.hotrod.configuration.ExecutorFactoryConfiguration;
+import org.infinispan.client.hotrod.configuration.NearCacheConfiguration;
+import org.infinispan.client.hotrod.configuration.SecurityConfiguration;
+import org.infinispan.client.hotrod.configuration.ServerConfiguration;
+import org.infinispan.client.hotrod.configuration.StatisticsConfiguration;
+import org.infinispan.client.hotrod.configuration.TransactionConfiguration;
+import org.infinispan.client.hotrod.impl.consistenthash.ConsistentHash;
+import org.infinispan.commons.configuration.BuiltBy;
+import org.infinispan.commons.marshall.Marshaller;
+import org.infinispan.commons.util.Features;
+
+@BuiltBy(ConfigurationBuilder.class)
+public class Configuration extends org.infinispan.client.hotrod.configuration.Configuration implements ClientConfig {
```

Comment:
-1 to extending `org.infinispan.client.hotrod.configuration.Configuration`.

Instead I would hide `org.infinispan.client.hotrod.configuration.Configuration` and delegate to an instance of that as needed. By extending it you're bringing 9 years worth of baggage onto a new API.

Reply:
perfect. it was for doing things fast but I'm definetly don't want to do this